### ISSUE TITLE
Should be able to write on to a masked field

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#19]
**Expected behavior**
With the write API the generated code should be able to write to a masked field(example a password field)

**Actual behavior**
Unable to write to a password field

**Steps to replicate**
Download [activeadmin-demo.war](https://bintray.com/artifact/download/gauge/activeadmin-demo/activeadmin-demo.war)
* java -jar activeadmin-demo.war
* start taiko
* Try the `log in` feature with password.

**Version**
```
Version: 0.0.9 (HeadlessChrome/64.0.3264.0)
```

**Comment by sguptatw** [comment]
Currently it is possible using the id selector of the password field. But not with the corresponding label name.

**Comment by NivedhaSenthil** [comment]
Works fine with the 7772f85

**Comment by sguptatw** [comment]
Unable to write to the password field. Here is the script to replicate the issue
```
> openBrowser()
 ✔ Browser opened
> goto("http://automationpractice.com/index.php")
 ✔ Navigated to url http://automationpractice.com/index.php
> click("Sign in")
 ✔ Clicked element matching text "Sign in"
> write("password",textBox(below("Password")))
 ✘ Error: Text field Below Password not found, run `.trace` for more info.
```
Taiko version
```
commit 3bb2ade397840ad2326026bb7fad840161e673ea
Version: 0.8.0 (Chromium: 74.0.3723.0)
```
Node version
```
v10.15.3
```

**Comment by Debashis9012** [comment]
Fixed as a part of #556
